### PR TITLE
Reimplement filter

### PR DESCRIPTION
**Pull request by CathalOConnorRH**
## Motivation
enabling the uri metrics may lead to excessive data, it would be nice to be able to limit/filter the uri to specific keywords

## What
added a URI_METRICS_FILTER_ENABLED flag to enable filtering and URI_METRICS_FILTER which uses a comma delimited list of keywords to filter. 

## Why
Add an short answer for: Why it was done? (E.g The feature X was deprecated.)

## How
Add an short answer for: How it was done? (E.g By removing this feature from ... OR By removing just the button but not its implementation ... ) 

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Go to `XX >> YY >> SS`
2. Create a new item `N` with the info `X`
3. Try to edit this item 
4. Check if in the left menu the feature X is not so long present.

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task
- [ ] TODO

## Additional Notes

PS.: Add images and/or .gifs to illustrate what was changed if this pull request modifies the appearance/output of something presented to the users.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
-keycloakVersion=12.0.1
+keycloakVersion=11.0.2
```
Comment (pb82) [inline gradle.properties:1]: Why was this updated?

**Comment by pb82** [comment]
@CathalOConnorRH it would be good to document those flags in the README

File: gradle.properties (hunk 1 above)
Comment (CathalOConnorRH) [reply gradle.properties]: I've reverted this

**Comment by CathalOConnorRH** [comment]
@pb82 Added flags to README and removed URI_METRICS_FILTER_ENABLED flag

**Review by pb82 — APPROVED** [review]

**Pull request merged**